src/assets/i18n/en.json

```diff
@@ -538,6 +540,7 @@
         "AVALANCHE_TRIGGER": "Sensitivity to triggers",
         "AVALANCHE_TYPE": "Avalanche type",
         "AVALANCHE_AREA": "Avalanche area",
+        "CONFIRM_AVALANCHE_AREA": "Confirm avalanche area",
```

Comment:
Jeg så ikke ulempen med å lage egne tekster for dette da jeg kommenterte det sist. Problemet er at jordskred ikke heter det samme som snøskred på engelsk. Så nå vises "Comfirm avalanche area" også for jordskred. Er redd vi må ha en CONFIRM_LANDSLIDE_AREA også..:)

Reply:
Aaah, selvfølgelig :)

Reply:
Grunnen til at jeg lagt inn alle språk er att disse tekstene i realiteten allerede fantes i appen, bare ikke i koden. Jeg lagde bare statiske strenger av det som allerede ble gjenerert. Men du mener at det vil ødelegge noe i språksynken?